src/db/z_db.erl

```diff
@@ -503,17 +540,33 @@ select(Table, Id, Context) ->
 
     Props = case Row of
         [R] ->
-            case proplists:get_value(props, R) of
-                PropsCol when is_list(PropsCol) ->
-                    lists:keydelete(props, 1, R) ++ PropsCol;
-                _ ->
-                    R
+            Props1 = case proplists:get_value(props, R, not_found) of
+                         undefined -> lists:keydelete(props, 1, R);
+                         PropsCol when is_list(PropsCol) ->
+                             lists:keydelete(props, 1, R) ++ PropsCol;
+                         _ -> R
+                     end,
+
+            case proplists:get_value(props_json, Props1, not_found) of
+                undefined -> lists:keydelete(props_json, 1, Props1);
+                Bin when is_binary(Bin) ->
+                    JP = json_decode(Bin),
+                    Normalised = [{maybe_to_atom(K), V} || {K, V} <- JP],
```

Comment:
This might be tricky - as it depends on the loaded modules which props are defined.  In the master we force more erlang modules (esp. models) to be loaded, so that more atoms are defined.  Maybe you need to do something similar here as well. Otherwise you might end up with atoms and binaries for the same keys, depending when they are loaded.